AirbnbSwiftFormatPlugin/AirbnbSwiftFormatPlugin.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+import PackagePlugin
+
+/// A Swift Package Manager `CommandPlugin` that executes `AirbnbSwiftFormatTool`
+/// to format source code in Swift package targets according to the Airbnb Swift Style Guide.
+@main
+struct AirbnbSwiftFormatPlugin: CommandPlugin {
+
+  func performCommand(context: PluginContext, arguments: [String]) async throws {
+    let process = Process()
+    process.launchPath = try context.tool(named: "AirbnbSwiftFormatTool").path.string
+
+    var processArguments = context.package.targets.map { $0.directory.string } + [
+      "--swift-format-path",
+      try context.tool(named: "swiftformat").path.string,
+      "--swift-lint-path",
+      try context.tool(named: "swiftlint").path.string,
+      // The process we spawn doesn't have read/write access to the default
+      // cache file locations, so we pass in our own cache paths from
+      // the plugin's work directory.
+      "--swift-format-cache-path",
+      context.pluginWorkDirectory.string + "/swiftformat.cache",
```

Comment:
What does this working directory look like? I want to see if it persists between plugin runs.

Reply:
The documentation says:
> The plugin is in complete control of what is written under this di- rectory, and the contents are preserved between builds.

On my machine this folder is `~/Documents/swift-style-guide/.build/plugins/AirbnbSwiftFormatPlugin/outputs`. The only two files inside it are the `swiftformat.cache` and `swiftlint.cache` files we create here.